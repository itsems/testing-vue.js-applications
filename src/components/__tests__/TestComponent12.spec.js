import { shallowMount } from '@vue/test-utils'
import Component from '../TestComponent12.vue'

describe('chapter-12 練習題', () => { 
  test('為此元件寫一個快照測試', () => { 
    const wrapper = shallowMount(Component)
    expect(wrapper.element).toMatchSnapshot()
   })
 })