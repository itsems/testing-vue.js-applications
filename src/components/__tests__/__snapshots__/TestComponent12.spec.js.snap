// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`chapter-12 練習題 為此元件寫一個快照測試 1`] = `
<div>
  Hello, World!
</div>
`;
